### PR TITLE
fix: use custom gptURL to verify key

### PR DESCRIPTION
**Pull request by GrayXu**

### CODE REVIEW

File: src/utils/gptModels.ts [hunk 1]
```diff
@@ -13,29 +14,9 @@ export async function gptStatusCallback(status: boolean) {
       const doc = dialog.window.document;
 
       try {
-        const models = await updateGPTModel();
```
Comment (windingwind) [inline src/utils/gptModels.ts:16]: Why is this removed?

Comment (GrayXu) [reply src/utils/gptModels.ts:16]: https://github.com/windingwind/zotero-pdf-translate/commit/868acc5f49a4abfef06e30aec634d82cfb8689a8 has removed the model selection for custom model input. As a result, the logic of `updateGPTModel()` has become redundant.

Comment (GrayXu) [reply src/utils/gptModels.ts:16]: But `authGPT()` still requests `/models` to verify the availability of `apiurl` and `key`. 

Perhaps by attempting a model chat request directly with limited tokens, it could provide more detailed error information. down to the model availability.  
However, different providers currently have inconsistent HTTP status codes in their responses. For instance, when a model does not exist, OpenAI returns a 404, while One-API returns a 503.

**Comment by windingwind** [comment]
Thanks. Since I no longer work on Zotero 6, I won't be able to test the z6 branch. If you confirm it's OK to merge, I'll push a new stable release.

**Comment by GrayXu** [comment]
Certainly, I have been testing it on Zotero 6 for a while now.
